Chocolatey.Cake.Recipe/Content/build.cake

```diff
@@ -74,6 +74,12 @@ BuildParameters.Tasks.CleanTask = Task("Clean")
     CleanDirectories(BuildParameters.Paths.Directories.ToClean);
 });
 
+BuildParameters.Tasks.InitTask = Task("Init")
+    .Does(() =>
```

Comment:
In the issue that is linked to this PR, there was mention of taking a dependency on the `Print-CI-Provider-Environment-Variables` task.  Did you decide that this wasn't a good idea?  If so, can you update the issue so that this requirement is removed, or, can you get the depedency added if it is required.

Reply:
I didn't decide that it wasn't a good idea, I just came to the conclusion that it was not necessary. It depending on `Print-CI-Provider-Environment-Variables` doesn't provide us anything when we run the task. Because we're never going to run it inside of CI (or I can't think of a place where we'll run it inside of CI). The actions that we want occur during setup and happen regardless of what we target. So I thought maybe just emitting a message that we're initializing would be just as impactful. If my memory is correct, this likely saves us at least 10 milliseconds by not running the task that doesn't output anything for us. I'll update the original issue to make note of this.

Reply:
I've updated the referenced issue with the suggestion that it just emits a message and let the projects add it to their dependency tree where they need it.